lib/modules/measure/MeasureService.ts

```diff
@@ -113,14 +113,14 @@ export class MeasureService extends BaseService {
      *
      * Useful to enrich measures before they are saved.
      */
-    await this.app.trigger<EventMeasureProcessSourceBefore>(
-      "device-manager:measures:process:sourceBefore",
+    await this.app.trigger<EventMeasureProcessBefore>(
```

Comment:
If i am not mistaken this is a breaking change ? 🤔

Reply:
This is for next major version so this is the time